server/apps/postgres-app/pom.xml

```diff
@@ -60,6 +60,24 @@
             <groupId>${james.groupId}</groupId>
             <artifactId>apache-james-mailbox-quota-search-scanning</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>blob-s3</artifactId>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>blob-s3-guice</artifactId>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-cassandra-app</artifactId>
```

Comment:
Suggestion: move S3extension into guice-s3 maven module? So that we do not need those testing dependencies onto `james-server-cassandra-app` and `james-server-distributed-app` ?

Reply:
I had same dependencies issues with some test classes depending on the cassandra app test scope

It might be a bit of a mess if we start relocating everything same time... A refactoring later maybe?

Reply:
Ok I buy the later refactoring as long as it is a ticket of this sprint ;-)

Reply:
https://github.com/linagora/james-project/issues/4999